src/server/exotics/movement-origin/type-of-product-location/index.test.js

```diff
@@ -0,0 +1,65 @@
+import { describePageSnapshot } from '~/src/server/common/test-helpers/snapshot-page.js'
+import { Answer, typeOfProductLocationPage } from './index.js'
+import { RadioButtonAnswer } from '~/src/server/common/model/answer/radio-button/radio-button.js'
+import { ProductLocationHasACphNumberPage } from '../product-location-has-a-cph-number/index.js'
+
+const sectionKey = 'movementOrigin'
+const questionKey = 'typeOfProductLocation'
+const pageUrl = '/exotics/movement-origin/product-location'
+const page = typeOfProductLocationPage
+const question = 'Where are the products located?'
+
+const payload = {
+  [questionKey]: 'some text'
```

Comment:
although it's not validated, I guess payload should technically have `{ [questionKey]: 'other' }` - or some other value that would realistically be sent?